<ui version="4.0" >
 <class>AdvancedPracticeOptions</class>
 <widget class="QWidget" name="AdvancedPracticeOptions" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>431</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>General Options</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="kcfg_AltLearn" >
        <property name="toolTip" >
         <string>By checking this, you will use the Leitner method of learning which requires you to answer correctly to each question 3 times in a row. </string>
        </property>
        <property name="whatsThis" >
         <string>By checking this, you will use the Leitner method of learning which requires you to answer correctly to each question 3 times in a row. </string>
        </property>
        <property name="text" >
         <string>&amp;Require 3 consecutive correct answers to count as right</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QCheckBox" name="kcfg_ShowMore" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="toolTip" >
           <string>Enable the Show More button on the Random Query screen</string>
          </property>
          <property name="whatsThis" >
           <string>When this is checked, you will be able to use a Show More button which allow you to get the next letter in your answer in the query. If this is unchecked, the Show More button will not be enabled, you will not be able to use it.</string>
          </property>
          <property name="text" >
           <string>Enable S&amp;how More button</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QCheckBox" name="kcfg_PracticeSoundEnabled" >
          <property name="text" >
           <string>Enable Sound</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QCheckBox" name="kcfg_SkipKnownEnabled" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="toolTip" >
           <string>Enable the I Know It button on the Random Query screen</string>
          </property>
          <property name="whatsThis" >
           <string>If this is checked, the I Know button will be available. It allows you to tell the query that you know the result without writing it or having it checked. This is available by default. If you uncheck this, the I Know button will not be available.</string>
          </property>
          <property name="text" >
           <string>Enable Skip (I Know it) &amp;button</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QCheckBox" name="kcfg_PracticeImagesEnabled" >
          <property name="text" >
           <string>Enable Images</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QCheckBox" name="kcfg_TestOrderLesson" >
          <property name="toolTip" >
           <string>When practicing new words will be chosen loosely in lesson order if this is selected, otherwise at total random.</string>
          </property>
          <property name="text" >
           <string>&amp;Practice in lesson order</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QCheckBox" name="kcfg_SwapDirection" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>S&amp;wap direction randomly</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QCheckBox" name="kcfg_CountSynonymsAsCorrect" >
          <property name="text" >
           <string>Accept synonyms as correct</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>&amp;Active entries during practice:</string>
          </property>
          <property name="buddy" >
           <cstring>kcfg_TestNumberOfEntries</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KIntSpinBox" name="kcfg_TestNumberOfEntries" >
          <property name="suffix" >
           <string/>
          </property>
          <property name="prefix" >
           <string/>
          </property>
          <property name="minimum" >
           <number>2</number>
          </property>
          <property name="maximum" >
           <number>10000</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>&amp;Show solution after answering (seconds):</string>
          </property>
          <property name="buddy" >
           <cstring>kcfg_TestNumberOfEntries</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KIntSpinBox" name="kcfg_ShowSolutionTime" >
          <property name="specialValueText" >
           <string>Off</string>
          </property>
          <property name="suffix" >
           <string/>
          </property>
          <property name="prefix" >
           <string/>
          </property>
          <property name="minimum" >
           <number>-1</number>
          </property>
          <property name="maximum" >
           <number>3600</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="PracticeTimeout" >
     <property name="toolTip" >
      <string>If a time limit is set, an answer has to be given in the set time, when practicing.</string>
     </property>
     <property name="title" >
      <string>Time Limit in Tests</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="KButtonGroup" name="kcfg_PracticeTimeout" >
        <property name="styleSheet" >
         <string>QGroupBox { border-width: 0; border-style: none; }</string>
        </property>
        <property name="title" >
         <string/>
        </property>
        <layout class="QVBoxLayout" >
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="NoTimeoutRadio" >
           <property name="toolTip" >
            <string>Check this if you do not want any time limitation per query.</string>
           </property>
           <property name="whatsThis" >
            <string>When this is checked, there is no time limitation for each query</string>
           </property>
           <property name="text" >
            <string>&amp;No time limitation</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="ShowSolutionRadio" >
           <property name="text" >
            <string>&amp;Show solution</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="ContinueRadio" >
           <property name="text" >
            <string>&amp;Continue after timeout</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_mqtime" >
            <property name="text" >
             <string>Ma&amp;x. time (s):</string>
            </property>
            <property name="buddy" >
             <cstring>kcfg_MaxTimePer</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <layout class="QHBoxLayout" >
            <item>
             <widget class="KIntSpinBox" name="kcfg_MaxTimePer" >
              <property name="toolTip" >
               <string>Set the maximum time allowed per answer.</string>
              </property>
              <property name="whatsThis" >
               <string>Set here the maximum time you want to allow per answer.</string>
              </property>
              <property name="minimum" >
               <number>1</number>
              </property>
              <property name="maximum" >
               <number>6000</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>569</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KButtonGroup</class>
   <extends>QGroupBox</extends>
   <header>kbuttongroup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>kcfg_SwapDirection</tabstop>
  <tabstop>kcfg_AltLearn</tabstop>
  <tabstop>kcfg_ShowMore</tabstop>
  <tabstop>kcfg_SkipKnownEnabled</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
